agreement_legal_sale/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Agreement Legal Sale',
+    'summary': 'Create an agreement when the sales order is confirmed',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'category': 'Agreement',
+    'website': 'https://github.com/OCA/contract',
+    'depends': [
+        'agreement_legal',
+        'sale_management',
```

Comment:
@murtuzasaleh I propose to create a module agreement_sale. 

Then instead of depends of sale_management you could depends of it.

Are you ok ?

I can make the PR today or tomorrow

cc @max3903

Reply:
@bealdav ok

Reply:
CC @murtuzasaleh https://github.com/OCA/contract/pull/309#issuecomment-482145205 ?